File: org.summer.view.widget/src/org/summer/view/widget/controls/GridUnitType.java
package org.summer.view.widget.controls;
/// <summary>
    /// GridUnitType enum is used to indicate what kind of value the 
    /// GridLength is holding. 
    /// </summary>
    // Note: Keep the GridUnitType enum in [....] with the string representation 
    //       of units (GridLengthConverter._unitString).
    public enum GridUnitType
    {
        /// <summary> 
        /// The value indicates that content should be calculated without constraints.
        /// </summary> 
        Auto ,//= 0, 
        /// <summary>
        /// The value is expressed as a pixel. 
        /// </summary>
        Pixel,
        /// <summary>
        /// The value is expressed as a weighted proportion of available space. 
        /// </summary>
        Star, 
    } 

File: summer/WebContent/js/system/windows/GridUnitType.js
/**
 * GridUnitType
 */
define(["dojo/_base/declare", "system/Type"], function(declare, Type){
	
    var GridUnitType = declare("GridUnitType", Object, { 
    });	
    
    GridUnitType.Auto = 0;
    GridUnitType.Pixel=1; 
    GridUnitType.Star=2; 

//    GridUnitType.Type = new Type("GridUnitType", GridUnitType, [Object.Type]);
    return GridUnitType;
});
